.circleci/config.yml

```diff
@@ -19,10 +19,18 @@ workflows:
           filters: *filters
       - shellcheck/check:
           filters: *filters
+      - orb-tools/publish:
```

Comment:
This should not happen in this file. This would publish before the testing passes. The next step `orb-tools/continue` kicks off the test-deploy which runs any test which need to pass before publishing. So just remove this block here.